config_test.go

```diff
@@ -467,3 +469,92 @@ func TestCustomFinder(t *testing.T) {
 		t.Errorf("expected to find User root, got %q", val)
 	}
 }
+
+func TestAliases(t *testing.T) {
+	testDir, err := os.MkdirTemp(os.TempDir(), "test-aliases")
+	if err != nil {
+		t.Fatal(err)
+	}
+	defer func() {
```

Comment:
Why not (?)

```suggestion
	defer os.RemoveAll(testDir)
```

Reply:
changed (that was because my dev environment shows a warning for not handling error).